docs/docs/cmd/teams/channel/channel-remove.md

```diff
@@ -30,28 +33,28 @@ When deleted, Microsoft Teams channels are moved to a recycle bin and can be res
 
 ## Examples
 
-Remove the specified Microsoft Teams channel by Id
+Remove the specified Microsoft Teams channel by Id _19:f3dcbb1674574677abcae89cb626f1e6@thread.skype_ from the Microsoft Teams team with id _d66b8110-fcad-49e8-8159-0d488ddb7656_
```

Comment:
I think keeping those description shorter and without the values used in the commands makes them actually a lot cleaner. I know many docs still follow the same approach but not so long ago we decided that this will make spec more clean.
what do you think? do you think we could follow the current (new 😉) approach ?